x-pack/plugins/aiops/public/hooks/use_data_source.ts

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { createContext, useContext } from 'react';
+import { DataView } from '@kbn/data-views-plugin/common';
+import { SavedSearch } from '@kbn/saved-search-plugin/public';
+import { SavedSearchSavedObject } from '../application/utils/search_utils';
+
+export const DataSourceContext = createContext<{
+  dataView: DataView | never;
+  savedSearch: SavedSearch | SavedSearchSavedObject | null;
+}>({
+  // @ts-ignore
```

Comment:
This ignore could do with a reason why

Reply:
fixed in https://github.com/elastic/kibana/pull/144093/commits/5db5d415564d7a4a92441a06029b937c81fad194